### ISSUE TITLE
See [my previous response](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+author%3ALayersOfAbstraction+)

### ISSUE THREAD

**Issue by LayersOfAbstraction** [dotnet/AspNetCore.Docs#27755]
> See [my previous response](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+author%3ALayersOfAbstraction+)

Hi @Rick-Anderson, firstly thanks for getting back to me on the weekend. Secondly, your previous response in #27695 was interpreted in such a way that it could mean: 

* That I make a response at the bottom of #26505 (which was marked as off-topic).
* That I go to the bottom of the page [Tutorial: Get started with EF Core in an ASP.NET MVC web app](https://learn.microsoft.com/en-us/aspnet/core/data/ef-mvc/?view=aspnetcore-6.0) (which was closed off)

The answer may be obvious to you but please understand you're reading from a person who is extremely new to contributing and has absolutely no IT commercial experience. 

In order to upgrade the majority of MVC documentation to .NET 6 it would be super helpful if you could provide a link to this doc page. I cannot find it.

**Comment by Rick-Anderson** [comment]
We plan on updating  the topic after #26505 is completed. Please stop opening new issues.

**Comment by Rick-Anderson** [comment]
you can ask follow up questions in a closed issue.

**Comment by LayersOfAbstraction** [comment]
> you can ask follow up questions in a closed issue.

Didn't know. Heard in blog posts that they can no longer be followed that's why I made a new one. Guess I was following misinformation. I'll wait until your dev team is finished with 26505.

**Comment by Rick-Anderson** [comment]
The dev team has that policy, but not the doc's team

**Comment by LayersOfAbstraction** [comment]
Just asking if I can do something unrelated to ASP.NET. I'm not sure who to ask and not sure if making another issue is ok or not. I was able to compress the images in my main portfolio using Calibre act

**Comment by LayersOfAbstraction** [comment]
Hi just asking if I can do something unrelated to ASP.NET. I'm not sure who to ask and not sure if making another issue is ok or not.

I notice DocFX doesn't support hot-reload. Some people have used an `npm-watch` to simulate a 'hot-reload' by re-triggering docfx build --serve when the file changes. 

I'm wondering if you guys would accept such a change if I were to implement it, please?
